ocs_ci/deployment/deployment.py

```diff
@@ -2007,7 +2007,6 @@ def deploy_with_external_mode(self):
             label_pod_security_admission(
                 namespace=constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE
             )
-            exec_cmd(f"oc create -f {constants.STORAGE_SYSTEM_ODF_EXTERNAL}")
```

Comment:
Shouldn't this change be applied conditionally only for 4.19 and above?

Reply:
done